File: src/main/java/axirassa/services/runners/PingerServiceRunner.java

package axirassa.services.runners;

import org.hornetq.api.core.client.ClientSession;

import axirassa.services.Service;
import axirassa.services.pinger.PingerService;
import axirassa.util.MessagingTools;

public class PingerServiceRunner {
	public static void main(String[] args) throws Throwable {
		ClientSession session = MessagingTools.getEmbeddedSession();
		Service service = new PingerService(session);
		
		service.execute();
	}
}

File: src/main/java/axirassa/services/runners/PingerServiceRunner.scala

package axirassa.services.runners

import axirassa.services.pinger.PingerService
import axirassa.util.MessagingTools

object PingerServiceRunner {
    def main(args : Array[String]) {
        val session = MessagingTools.getEmbeddedSession()
        val service = new PingerService(session)

        service.execute()
    }
}
